### ISSUE TITLE
Compiling on non RHEL related distro

### ISSUE THREAD

**Issue by EmmaTheDevTDGZ** [Unidata/awips2#675]
Howdy folk.

I was wondering if anyone here can direct me to any resources relating to compiling this on a non-RHEL related distro or a place I can retrieve binaries for using this on a different linux distro.

**Comment by srcarter3** [comment]
Hi there, the only way we have built CAVE for other operating systems (for Windows and for MacOS), is to import all the source code from all the repos into Eclipse and export from Eclipse and build installers that way.
Is this what you're looking to do?

**Comment by EmmaTheDevTDGZ** [comment]
Yes I could probably do that if needed. Is all of the source code that is required to be imported into eclipse located within this one repo or is it multiple?

**Comment by srcarter3** [comment]
Will rpms work on the OS you're wanting to create a CAVE installation for?
If so, we may have something that can set up the proper development environment (correct versions of Python, Java, and Eclipse).  If not, I can give you the versions and you'll manually have to setup and install them.

And there are several repos necessary.

Beginning at [Step 4 on our development instructions webpage](https://unidata.github.io/awips2/dev/awips-development-environment/#4-download-the-source-code), these should be mostly accurate for how to set up eclipse.

I don't think we've updated this page since transitioning from v18 to v20 of AWIPS, but they are still pretty accurate (besides the version numbers of java, python, and eclipse).

The correct versions are:
- Eclipse: 4.17.0
- Java: 11.0.13
- Python: 3.6

Let me know if that makes sense?

**Comment by EmmaTheDevTDGZ** [comment]
Yeah that makes sense, I'll attempt putting it all together later today hopefully.
